src/jit/valuenum.cpp

```diff
@@ -5162,7 +5213,8 @@ void Compiler::fgValueNumberTree(GenTreePtr tree, bool evalAsgLhsInd)
                                 assert(typSize > varSize);
                                 // the indirection is reading beyond the end of the field
                                 //
-                                lcl->gtVNPair.SetBoth(vnStore->VNForExpr(compCurBB, typ)); // return a new unique value number
+                                lcl->gtVNPair.SetBoth(vnStore->VNForExpr(compCurBB, typ)); // return a new unique value
```

Comment:
All the formatting got messed up on one of my merges, so I re-ran jit-format. So there are some formatting-only changes mixed in here.

---
In reply to: [77282193](https://github.com/dotnet/coreclr/pull/7027#discussion_r77282193) [](ancestors = 77282193)